presto-bytecode/src/main/java/com/facebook/presto/bytecode/MethodDefinition.java

```diff
@@ -30,13 +31,16 @@
 import static com.facebook.presto.bytecode.Access.STATIC;
 import static com.facebook.presto.bytecode.Access.toAccessModifier;
 import static com.facebook.presto.bytecode.ParameterizedType.type;
+import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.collect.Iterables.transform;
 import static org.objectweb.asm.Opcodes.RETURN;
 
 @SuppressWarnings("UnusedDeclaration")
 @NotThreadSafe
 public class MethodDefinition
 {
+    private static final int MAX_ALLOWED_VARARG_ARGUMENTS = 250;
```

Comment:
The maximum number of parameters for a method is 255: https://docs.oracle.com/javase/specs/jvms/se8/html/jvms-4.html#jvms-4.3.3

No need for this constant, as it's well known and specified as part of the JVMS.